edi_storage_oca/utils.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+import base64
+import os
+import re
+
+
+def add_file(storage, path, filedata, binary=False):
```

Comment:
nice. This is something that should probably be proposed to the base module as I don't see any edi specific stuff.

Reply:
It was in the base module `fs_storage` before (as Odoo methods, not functions), but deprecated. Thing is we still need to encapsulate such features somewhere yes.